### PR TITLE
Create README.md

### PR DESCRIPTION
**Pull request by rajithaBK**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+WEBEX_ACCESS_TOKEN=
+WEBEX_ORG_ID=
+WEBEX_TEAM_ID=
+WEBEX_JWT_SECRET=
+PORT=
+
+```
+Note:
+1. You will need to provide a PORT for this to run locally, or else if empty, it runs on port 3000
+2. You will need to provide an WEBEX_ACCESS_TOKEN of a test bot for testing
```
Comment (askmrsinh) [inline README.md]: Link to instructions for creating a Webex bot. Also, use monospace font for `WEBEX_ACCESS_TOKEN `

File: README.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+3. You will need to create a new Guest Issuer application in My Webex Apps and provide the WEBEX_ORG_ID and WEBEX_JWT_SECRET.
```
Comment (askmrsinh) [inline README.md]: Link to instructions for getting these details. Also, use monospace font formatting for env variables.

File: README.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+4. You will need to provide WEBEX_TEAM_ID in which the spaces are created.
```
Comment (askmrsinh) [inline README.md]: Again, link to instructions, example team id.

File: README.md [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+and the response contains the response URL, which looks like:
+```
+{
+    "redirect": ""
+}
+
+```
+
+### Setup
+You will need to create a file called **.env** that includes the following lines:
```
Comment (askmrsinh) [inline README.md]: Include instructions to copy paste the `env.example` file:

```
cp .env.example .env
```

File: README.md [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+        "responder":"",
+        "requester":""
+    }
+}'
+
+```
+and the response contains the response URL, which looks like:
+```
+{
+    "redirect": ""
```
Comment (askmrsinh) [inline README.md]: Add dummy redirect url as example.

File: README.md [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+### Usage
+One can use the Mindy Bot by sending a post request to 
+```
+curl --location --request POST 'https://mindy.wbx.ninja/virtual-nurse-request' \
+--header 'Content-Type: application/json' \
+--data-raw '{
+    "guid":"",
+    "labels":{
+        "responder":"",
+        "requester":""
```
Comment (askmrsinh) [inline README.md]: Add example values.

File: README.md [hunk 7]
```diff
@@ -0,0 +1,52 @@
...
+Note:
+1. You will need to provide a PORT for this to run locally, or else if empty, it runs on port 3000
+2. You will need to provide an WEBEX_ACCESS_TOKEN of a test bot for testing
+3. You will need to create a new Guest Issuer application in My Webex Apps and provide the WEBEX_ORG_ID and WEBEX_JWT_SECRET.
+4. You will need to provide WEBEX_TEAM_ID in which the spaces are created.
+
+### Install
+The typical npm install flow, after cloning this repo
+```
+npm init
```
Comment (askmrsinh) [inline README.md]: Why do we need `npm init`?

File: README.md [hunk 8]
```diff
@@ -0,0 +1,52 @@
...
+
+```
+Note:
+1. You will need to provide a PORT for this to run locally, or else if empty, it runs on port 3000
+2. You will need to provide an WEBEX_ACCESS_TOKEN of a test bot for testing
+3. You will need to create a new Guest Issuer application in My Webex Apps and provide the WEBEX_ORG_ID and WEBEX_JWT_SECRET.
+4. You will need to provide WEBEX_TEAM_ID in which the spaces are created.
+
+### Install
+The typical npm install flow, after cloning this repo
```
Comment (askmrsinh) [inline README.md]: Add instructions to do a git clone.

**Review by askmrsinh — CHANGES_REQUESTED** [review]

**Comment by rajithaBK** [comment]
Ill make changes and create a new pull request with additonal cors changes

File: README.md [hunk 9]
```diff
@@ -0,0 +1,75 @@
...
+NINJA_API_URL=https://wxsd.wbx.ninja
+SOAPBOX_API_URL=https://soapbox.wbx.ninja
+WEBEX_ACCESS_TOKEN=
+WEBEX_ORG_ID=
+WEBEX_TEAM_ID=
+WEBEX_JWT_SECRET=
+PORT=
+```
+
+Review and follow the [Quickstart: Register an application with the Microsoft identity platform](https://docs.microsoft.com/en-us/azure/active-directory/develop/quickstart-register-app#register-an-application)
```
Comment (askmrsinh) [inline README.md]: We don't need this.

File: README.md [hunk 10]
```diff
@@ -0,0 +1,75 @@
...
+        "requester":"Requester Nurse"
+            }
+}'
+```
+
+and the response contains the response URL, which looks like:
+
+```
+{
+    "redirect": "https://wxsd.wbx.ninja/guest?headerToggle=false&destination=Y2lzY29zcGFyazovL3VzL1JPT00vNWQxYjU0MTAtMDg2My0xMWVkLTkzZDItOTllNWQ0Mm&userType=licensed&token=YTFlM2Q3NjQtMDYxMC00YzM1LWFjODktNDNlOWM1OTc5MGRlNTk5ZTA5NmEtMTU5_PF84_578771dd-81f7-42d6-9dbb"
```
Comment (askmrsinh) [inline README.md:26]: I hope these are dummy values and on't point to actual space / token?

**Review by askmrsinh — CHANGES_REQUESTED** [review]

**Review by askmrsinh — APPROVED** [review]

**Pull request merged**